pipelines/crawler/bndes/constants.py

```
@@ -0,0 +1,148 @@
+        "setor_cnae": "setor_cnae",
+        "subsetor_cnae_agrupado": "subsetor_cnae_agrupado",
+        "subsetor_cnae_codigo": "codigo_subsetor_cnae",
+        "subsetor_cnae_nome": "nome_subsetor_cnae",
+        "setor_bndes": "setor_bndes",
+        "subsetor_bndes": "subsetor_bndes",
+        "porte_do_cliente": "porte_cliente",
+        "natureza_do_cliente": "natureza_cliente",
+        "instituicao_financeira_credenciada": "instituicao_financeira_credenciada",
+        "cnpj_do_agente_financeiro": "cnpj_agente_financeiro",
+        "situacao_da_operacao": "situacao_operacao",
+    }
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)